docs/RESOURCES.md

```diff
@@ -1,20 +1,12 @@
 # Resources
 
-To get help if you're having trouble, you can use one of the following resources:
+- Learning resources
+    - [Get Started](https://ballerina.io/learn/get-started/)
+    - [Ballerina By Example](https://ballerina.io/learn/by-example/)
+    - [Learn Ballerina](https://ballerina.io/learn/)
+    - [Ballerina specifications](https://ballerina.io/learn/ballerina-specifications/)
+    - [Ballerina API Docs](https://lib.ballerina.io/)
+- Community channels    
```

Comment:
Shall we add a section for Twitter and LinkedIn pages too.

Reply:
Added.